test/testapps/BasicTestApp/TouchEventComponent.cshtml

```diff
@@ -0,0 +1,38 @@
+@using System.Collections.Generic
+
+<div>
+    <h2>Touch position</h2>
+    <p>
+        Output: <span id="output">@message</span>
+    </p>
+    <p>
+        <button ontouchstart=@OnTouch
+                ontouchcancel=@OnTouch
+                ontouchenter=@OnTouch
+                ontouchleave=@OnTouch
+                ontouchend=@OnTouch
+                ontouchmove=@OnTouch>
+            TOUCH ME
+        </button>
+    </p>
+    <p>
+        <button onclick="@Clear">Clear</button>
+    </p>
+</div>
+
+@functions {
+
+    string message;
+
+    void OnTouch(UITouchEventArgs e)
+    {
+        message += e.Type;
+        Console.WriteLine(JsonUtil.Serialize(e));
+        StateHasChanged();
+    }
+
+    void Clear()
+    {
+        message = string.Empty;
+    }
```

Comment:
Would it be possible to write an E2E test for this? Similar to https://github.com/aspnet/Blazor/blob/dev/test/Microsoft.AspNetCore.Blazor.E2ETest/Tests/EventTest.cs#L91

You should be able to use https://seleniumhq.github.io/selenium/docs/api/dotnet/html/T_OpenQA_Selenium_Interactions_TouchActions.htm